resources/web/wwi/Makefile

```diff
@@ -39,8 +40,27 @@ BABEL_JS_SOURCES = \
   node_modules/@babel/polyfill/dist/polyfill.js
 
 DEPENDENCIES_PATH = dependencies
+THEEEJS_VERSION = 104
 THREEJS_SOURCE = $(DEPENDENCIES_PATH)/three.min.js
-THREEJS_URL = https://cdnjs.cloudflare.com/ajax/libs/three.js/104/three.min.js
+THREEJS_URL = https://cdnjs.cloudflare.com/ajax/libs/three.js/$(THEEEJS_VERSION)/three.min.js
+
+# external sources to be downloaded and minified
+THREEJS_EXAMPLE_URL = https://cdn.jsdelivr.net/gh/mrdoob/three.js@r$(THEEEJS_VERSION)/examples/js/
+THREEJS_EXAMPLE_SOURCES = \
+  loaders/RGBELoader.js \
+  pmrem/PMREMCubeUVPacker.js \
+  pmrem/PMREMGenerator.js \
+  shaders/CopyShader.js \
+  shaders/DepthLimitedBlurShader.js \
+  shaders/FXAAShader.js \
+  shaders/GammaCorrectionShader.js \
+  shaders/SAOShader.js \
+  shaders/UnpackDepthRGBAShader.js \
+  postprocessing/EffectComposer.js \
+  postprocessing/RenderPass.js \
+  postprocessing/SAOPass.js \
```

Comment:
The dependency list should be minimized (it contains several unnecessary files for this PR, like the SAO file).